src/common/util/src/file_util.cpp

```diff
@@ -416,6 +422,27 @@ void ov::util::create_directory_recursive(const std::string& path) {
     }
 }
 
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+void ov::util::create_directory_recursive(const std::wstring& path) {
+    if (path.empty() || directory_exists(path)) {
+        return;
+    }
+
+    std::size_t pos = path.rfind(ov::util::FileTraits<char>::file_separator);
```

Comment:
Should it be wstring?

Reply:
you mean the "file_separator" should be wstring?